examples/bridge-app/bridge-common/gen/call-command-handler.cpp

```diff
@@ -350,14 +350,14 @@ EmberAfStatus emberAfOnOffClusterServerCommandParse(EmberAfClusterCommand * cmd)
             {
                 return EMBER_ZCL_STATUS_MALFORMED_COMMAND;
             }
-            onOffControl = emberAfGetInt8u(cmd->buffer, payloadOffset, cmd->bufLen);
-            payloadOffset += 1;
+            onOffControl  = emberAfGetInt8u(cmd->buffer, payloadOffset, cmd->bufLen);
+            payloadOffset = static_cast<uint16_t>(payloadOffset + 1);
```

Comment:
does `payloadOffset++` work instead?

Reply:
To circle back to this: due to C's integer-promotion rules, which C++ inherits, no.